src/System.Net.Http/src/netcore50/project.json

```diff
@@ -16,6 +16,7 @@
         "System.Resources.ResourceManager": "4.0.0",
         "System.Runtime.Extensions": "4.0.10",
         "System.Runtime.WindowsRuntime": "4.0.10",
+        "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-24008-00",
```

Comment:
@davidsh, I believe @bartonjs is saying it's fine to take this dependency for RTM.  @bartonjs, correct?